### PR TITLE
reviewed the wording & flow

### PR DESCRIPTION
**Pull request by avermeulen**
@SteveBarnett please have a look and merge if you are happy.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Log in to [GitHub](http://github.org) and fork this [repository](https://github.
 
 ### Look at it
 
-You should write all your code in `app.js`.
+You should write all your code in `app.js`. The `support.js` file contains some helper functions that has been written for you, don't edit this file! All the HTML & CSS are supplied.
```
Comment (SteveBarnett) [inline index.md]: The `support.js` file contains some helper functions that has been written for you, don't edit this file! All the HTML & CSS are supplied.

->

The `support.js` file contains some helper functions that have been written for you. Don't edit this file! All the HTML & CSS is supplied: don't edit these either!

File: instructions/first_function.md [hunk 2]
```diff
@@ -4,35 +4,33 @@ layout: default
...
 
-To capture keystroke we need to add an event handler (function) to the body of the html document's `onkeydown` event.
+Open the [unit test](http://localhost:8080/tests.html) page and make sure the `keyCodeName` tests are passing. You'll need to refresh the page if you already have it open in your browser.
+
+## Capture key codes
+
+To capture keystrokes we need to add an event handler (function) to the body of the html document's `onkeydown` event.
 
-You can use the display function to display the keycode on the screen `displayMessage(e.keyCode);`
+The nice people of codeX have created some functions for you in the `support.js` file to make your live easier. It has already been included for you. One of them is `displayMessage` which you can use to display a message on the web page.
```
Comment (SteveBarnett) [inline instructions/first_function.md:28]: live -> life

File: instructions/move_the_taxi.md [hunk 3]
```diff
@@ -6,18 +6,22 @@ layout: default
 
 ![](/img/move-the-taxi.jpg)
 
-The taxi should move forward when the right arrow is pressed. The taxi's position is determined by a class attached to it. There are nine classes: `.slot-one-of-nine` to `.slot-nine-of-nine`. Attach the next class to the taxi elements as the forward key is pressed.
+The taxi should move forward when the right arrow is pressed. The taxi's position is determined by a class attached to it. There are nine classes: `.slot-one-of-nine` to `.slot-nine-of-nine`. Attach the next class to the taxi element as the forward key is pressed.
 
-The algorithm you will use to move the taxi forward is:
+Here's the algorithm will use to move the taxi forward. We will go through is step by step.
```
Comment (SteveBarnett) [inline instructions/move_the_taxi.md]: through is -> through it

File: instructions/move_the_taxi.md [hunk 4]
```diff
@@ -6,18 +6,22 @@ layout: default
...
-    2. when the forward key is pressed increment `taxiLocationCounter` by 1;
-    3. when the back key is pressed decrement `taxiLocationCounter` by 1;
-    4. make sure that the counter variable never get's bigger than 9 or less than 1;
-    5. translate the counter's value into class names;
-    6. when the forward key is pressed, replace the old location class with the new location class on the taxi html element.
+The algorithm:
 
-Go ahead and implement steps 1 to 4. Display the value of `taxiLocationCounter` on the screen using the `displayMessage` function.
+    1. initialize a global variable (declare it outside `onkeydown`) called `taxiLocationCounter` to 1;
+    * when the forward key is pressed increment `taxiLocationCounter` by 1;
```
Comment (SteveBarnett) [inline instructions/move_the_taxi.md]: is pressed -> is pressed,

File: instructions/move_the_taxi.md [hunk 5]
```diff
@@ -6,18 +6,22 @@ layout: default
...
+    * when the back key is pressed decrement `taxiLocationCounter` by 1;
```
Comment (SteveBarnett) [inline instructions/move_the_taxi.md]: is pressed -> is pressed,

File: instructions/move_the_taxi.md [hunk 6]
```diff
@@ -6,18 +6,22 @@ layout: default
...
+    * make sure that the counter variable never get's bigger than 9 or less than 1;
```
Comment (SteveBarnett) [inline instructions/move_the_taxi.md]: get's -> gets

**Comment by SteveBarnett** [comment]
@avermeulen Looks p good :). I've added a few tiny comments.

**Comment by avermeulen** [comment]
@SteveBarnett no more conflits now. Your comments are :+1:

**Comment by SteveBarnett** [comment]
Bueno! :+1:

**Pull request merged**